bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/config/BridgeConfiguration.java

```diff
@@ -31,4 +31,5 @@ public class BridgeConfiguration {
     public int bufferApiCommands = 450;
     public int pollingInterval = 0;
     public int pollingIntervalErrors = 60;
+    public Boolean disablePolling = false;
```

Comment:
```suggestion
    public boolean disablePolling = false;
```

Use primitives where possible.